src/style.css

```
@@ -0,0 +1,35 @@
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.any__star {
+  background: url(images/star.svg) no-repeat center;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 .any__star:first-child {
```

Use this nth-of-type, as well
```suggestion
.stars--1 .any__star:first-child {
```